docs/aad-e2e/aad-e2e.md

```diff
@@ -0,0 +1,234 @@
+### Deploy OpenPAI depending on Azure Active Directory
+
```

Comment:
Start from version *** OpenPAI can be configured to use Azure Active Directory (AAD) for user authentication. In this configuration, you can sign in to an OpenPAI cluster by using your AAD authentication token.

This document will cover how to set up the integration step by step.

Reply:
do you want to add any important "Note" at the beginning? i.e. AAD will replace the user management, it is not an co-exisit, etc.

Reply:
Done